frontend/src/components/ViewActivities/index.js

```diff
@@ -1,14 +1,21 @@
 import React from 'react';
+import ActivityList from './activitylist';
 
-/**
- * ViewActivities component.
- */
-const ViewActivities = () => {
-  return (
-    <div>
-      <h1>View Activities</h1>
-    </div>
-  );
-};
+class ViewActivities extends React.Component {
+  /** @inheritdoc */
+  constructor(props) {
+    super(props);
+    this.state = {tripId: props.match.params.tripId};
+  }
+
+  /** @inheritdoc */
+  render() {
+    return (
+      <div className="activity-page"> 
```

Comment:
Preference thing so this is up to you. But I think you can remove this div (less nested divs in the end result) and transfer all the css rules for that class name to rules on the ActivityList tag.

Reply:
I was trying to do a body HTML tag and it didn't really go as planned. This was the alternative solution. 
Any suggestions?

Reply:
A body tag? Isn't that only used once at the top level? I was suggesting to remove the div and then have something like this in your CSS (but I am not 100% sure you can do this):

ActivityList {
    display: flex;
    flex-direction: column;
    ...
}

Reply:
Exactly. I wanted top level. If you look at .activity-page on activities.css, there's a lot of setup being done regarding the general layout the of the page.